verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -533,9 +546,15 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
                                 }
                             >
                                 {this.tableOfContentsIcon({
-                                    isChecked: values.travelHistory?.length > 0,
+                                    isChecked:
+                                        values.travelHistory?.length > 0 ||
```

Comment:
I don't think you need the .length > 0 here if you don't access the array after that

Reply:
The initial value of travelHistory is [] so we want to check if there are any entries in the array. This is currently an unnecessary check since the array will always be [] if `traveledPrior30Days === undefined`, but there is a bug out to change the criteria for a check in the table of contents, so I will leave it here until that changes.